ieg-european-history-en.csl

```diff
@@ -0,0 +1,367 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" page-range-format="expanded" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>IEG-2-en</title>
```

Comment:
Please input the full, written out title here.
`IEG-2` could go into title-short if necessary. Just below.